services/hanke-service/src/integrationTest/kotlin/fi/hel/haitaton/hanke/hakemus/HakemusServiceITest.kt

```diff
@@ -1178,7 +1184,7 @@ class HakemusServiceITest(
                             .prop(Hakemusyhteystieto::yhteyshenkilot)
                             .extracting { it.hankekayttajaId }
                             .containsExactlyInAnyOrder(kayttaja.id, newKayttaja.id)
-                        prop(KaivuilmoitusData::areas).isNotNull().single().isEqualTo(area)
+                        prop(KaivuilmoitusData::areas).isNotNull().single()
```

Comment:
Since the request no longer contains tormaystarkastelutulos, we cannot check the equality. For this test I think it is enough to check that there is a single area.

Reply:
Fixed this by overloading equals and hashCode of Tyoalue to exclude Tormaystarkastelutulos. Now this can be done as it was with isEqualTo(area).